README.rst

```diff
@@ -1,65 +1,60 @@
+* Download an `OpenVPN installer <https://openvpn.net/index.php/open-source.html>`_
+* If you have a previous version of OpenVPN GUI running, shut it down.
   Make sure it's closed by ALL logged on users.
 
-* Run the install program. During the installation you can choose if the GUI
-  should be started automatically at system startup. The default is yes.
-
-* Create a xxxx.ovpn config-file with your favorite texteditor and save it in:
-  C:\Program files\OpenVPN\config\. You should NOT use the "log" or "log-append"
-  options as OpenVPN GUI redirect the normal output to a log file itself. 
-  There is a sample config files in the "sample-config" folder. Please
-  refer to the OpenVPN project homepage for more information regarding 
-  creating the configuration file. http://openvpn.net/
-
+* Run the OpenVPN installer
 
 Manual installation of OpenVPN GUI
 **********************************
 
-* Download and install OpenVPN from http://openvpn.net/
+* `Download <https://openvpn.net/index.php/download/community-downloads.html>`_
+  and install OpenVPN
 
-* Download openvpn-gui-1.0.X.exe and save it in OpenVPN's bin folder.
-  Default is "C:\Program Files\OpenVPN\bin\". You must put it in this folder
+* Download OpenVPN GUI of your choice and save it in OpenVPN's bin folder.
+  Default is *C:\\Program Files\\OpenVPN\\bin\\*. You must put it in this folder
   because OpenVPN GUI depends on the OpenSSL DLLs installed in this folder by
   OpenVPN.
 
-* Create a xxxx.ovpn config-file with your favorite texteditor and save it in:
-  C:\Program files\OpenVPN\config\. You should NOT use the "log" or "log-append"
-  options as OpenVPN GUI redirect the normal output to a log file itself. 
-  There is a sample config files in the "sample-config" folder. Please
-  refer to the OpenVPN project homepage for more information regarding 
-  creating the configuration file. http://openvpn.net/
+Configuring OpenVPN GUI to start on Windows logon
+*************************************************
+
+Put a short-cut to openvpn-gui.exe in your *Start -> All Program -> StartUp*
+folder if you want the gui to start automatically when you logon to Windows.
 
-* Put a short-cut to openvpn-gui-1.0-X.exe in your 
-  "Start->All Program->StartUp" folder if you want the gui started automatically
-  when you logon to Windows.
+Adding an OpenVPN configuration file
+************************************
 
-* Start the GUI by double-clicking the openvpn-gui-1.0.X.exe file. You need to 
-  be Administrator the first time you run OpenVPN GUI for it to create its 
-  registry keys. After that you don't have to be administrator just to run the 
-  GUI, however OpenVPN requires the user to be administrator to run!
+To launch a VPN connections using OpenVPN GUI you must add an OpenVPN
+configuration file with .ovpn suffix and put it to *C:\\Program
+Files\\OpenVPN\config\\*. Any text editor (e.g. notepad.exe) can be used to
```

Comment:
- The config could be in C:\Users\username\OpenVPN\config or any of its immediate subdirectories as well
- A config file could be imported via the "Import file" menu  
So the "must add  a config to C:\Progra..." statement could be relaxed.